src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -48,7 +61,15 @@ public String findElement(List<String> elements, String element) {
      * NoSuchElementException</p>
      **/
     public Double averageSumOdd(List<Integer> numbers) {
-        return null;
+        for (int i = 1; i < numbers.size(); i += 2) {
```

Comment:
@SviatoslavExpert Can you use `.map` functions in Stream API instead of this loop?